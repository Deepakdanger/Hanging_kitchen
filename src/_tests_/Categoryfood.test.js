import renderer from 'react-test-renderer';
import { render } from '@testing-library/react';
import { StaticRouter } from 'react-router-dom';
import Catagoryfood from '../components/Categoryfood';

describe('Testing Filter with static router', () => {
  it('Categoryfood component is defined', async () => {
    const component = render(
      <StaticRouter>
        <Catagoryfood ele={{}} selectFood={jest.fn()} />
      </StaticRouter>,
    );
    expect(component).toBeDefined();
  });

  it('Categoryfood snapshot testing', () => {
    const categoryfood = renderer.create(
      <StaticRouter>
        <Catagoryfood ele={{}} selectFood={jest.fn()} />
      </StaticRouter>,
    ).toJSON();
    expect(categoryfood).toMatchSnapshot();
  });
});
