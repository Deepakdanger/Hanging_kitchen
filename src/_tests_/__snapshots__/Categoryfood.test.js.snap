// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Filter with static router Categoryfood snapshot testing 1`] = `
<div
  className="meals_dish swing"
>
  <div
    className="meals_img_div"
  >
    <img
      alt="NA"
      className="meals_img"
    />
  </div>
  <a
    href="/food"
    onClick={[Function]}
  >
    <button
      className="category_name"
      onClick={[MockFunction]}
      type="button"
    />
  </a>
</div>
`;
